pkg/health/health.go

```diff
@@ -66,18 +69,30 @@ func (s *ServiceNodeStatus) IsCurrentVersion(version string) bool {
 
 type ConsulHealthChecker struct {
 	Health *consulapi.Health
+	Store  kp.Store
 }
 
-func NewConsulHealthChecker(consulHealth *consulapi.Health) *ConsulHealthChecker {
+func NewConsulHealthChecker(store kp.Store, consulHealth *consulapi.Health) *ConsulHealthChecker {
 	return &ConsulHealthChecker{
 		Health: consulHealth,
+		Store:  store,
 	}
 }
 
-func (s *ConsulHealthChecker) toNodeStatus(entry consulapi.ServiceEntry) *ServiceNodeStatus {
+func (s *ConsulHealthChecker) toNodeStatus(serviceID string, entry consulapi.ServiceEntry) *ServiceNodeStatus {
+
+	version := UnknownSHA
+	manifest, _, err := s.Store.Pod(kp.RealityPath(entry.Node.Node, serviceID))
+	if manifest != nil && err == nil {
```

Comment:
well, more specifically, we should probably know if this fails, either by returning the error or something smarter.